modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/Layout/context.tsx

```diff
@@ -22,6 +22,7 @@ import {BoxesVisitor} from '../../utils/visitor';
 import {TObjectField, TObjectLayout, TObjectRelationship} from './types';
 
 type TState = {
+	ffMetadataFieldsAsSystemFields: boolean;
```

Comment:
@danvb112 You don't need to receive the value for the Feature Flag from the back-end and pass it through the components. You could just use `Liferay.FeatureFlags['LPS-154872']` :)